tests/integration/testlib/assertlib.py

```diff
@@ -35,13 +36,38 @@ def assert_state(desired_state_data):
     assert desired_state.state == current_state.state
 
 
-def assert_absent(*ifnames):
+def assert_absent(*ifnames, timeout=0):
     """ Assert that a interface is not present in the current state """
 
-    current_state = statelib.show_only(ifnames)
+    now = time.time()
+    while True:
```

Comment:
I think I suggested a decorator in the original PR.
Can we do that?

(it is already duplicated here in two asserts and probably other asserts will need it in the future)